js/libraries/jbox/jBox.css

```diff
@@ -19,7 +19,6 @@
 .jBox-container {
 	background: #fff;
 	border:1px solid #ffbb00;
-	max-width:180px;
```

Comment:
All good then, and let's try and remember to patch again if we ever update jBox. :-P